x/programs/rust/sdk-macros/src/lib.rs

```diff
@@ -258,7 +261,7 @@ pub fn state_keys(_attr: TokenStream, item: TokenStream) -> TokenStream {
 
     // add default attributes
     item_enum.attrs.push(parse_quote! {
-         #[derive(Clone, Copy, Debug, Eq, PartialEq, Hash)]
+         #[derive(Clone, Copy, Debug, Eq, PartialEq, Hash, wasmlanche_sdk::bytemuck::NoUninit)]
```

Comment:
What does the error message look like if `bytemuck` isn't imported? What if it *is* imported, but is missing the `derive` feature?

Reply:
Actually... this might not be a relevant change... maybe back this one off